[Bug] Compile error: duplicated destructor function in MinMaxFuncBase

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.


### Version

master

### What's Wrong?

```
incubator-doris/be/src/exprs/minmax_predicate.h:38:13: error: 'virtual doris::MinMaxFuncBase::~MinMaxFuncBase()' cannot be overloaded with 'virtual doris::MinMaxFuncBase::~MinMaxFuncBase()'
```

### What You Expected?

compile success

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/be/src/exprs/minmax_predicate.h
+++ b/be/src/exprs/minmax_predicate.h
@@ -35,8 +35,6 @@ public:
     virtual Status assign(void* min_data, void* max_data) = 0;
     // merge from other minmax_func
     virtual Status merge(MinMaxFuncBase* minmax_func, ObjectPool* pool) = 0;
-    virtual ~MinMaxFuncBase() = default;
-    ;
 };
 
 template <class T>

